internal/k8s/serialize_test.go

```diff
@@ -91,12 +91,23 @@ func TestCRDYAML(t *testing.T) {
 		"projects.example.martin-helmich.de",
 		"example-project",
 	}, names)
+}
+
+func TestPodDisruptionBudgetYAML(t *testing.T) {
```

Comment:
add a comment about what behavior this is supposed to test?

Reply:
done!